pkg/verifier/cosign/cosign.go

```diff
@@ -295,7 +295,7 @@ func (v *cosignVerifier) verifyInternal(ctx context.Context, subjectReference co
 		), nil
 	}
 
-	errorResult := errorToVerifyResult(v.name, v.verifierType, fmt.Errorf("no valid signatures found"))
+	errorResult := errorToVerifyResult(v.name, v.verifierType, fmt.Errorf("No valid Cosign signatures found"))
```

Comment:
Would it be better to use lower case according to the [go doc](
https://go.dev/wiki/CodeReviewComments#error-strings:~:text=multiple%20return%20values.-,Error%20Strings,-%C2%B6)